elastic/integrations
# [microsoft_sqlserver] Add transaction log DataStream

What does this PR do?

Collects the transaction log metrics for Microsoft Sql server. 

Please refer this [PR](https://github.com/elastic/integrations/pull/3391) for performance datastream.

- Added a data stream (transaction_log) under microsoft_sqlserver integration package
-  Added support to collect metrics from Named Instance.
- Added data collection logic for the data streams.
- Added fields metadata in the appropriate yaml files.
- Added dashboards and visualisations.
- Updated documentation for the datastreams



## Checklist





MSSQL server integration using generic sql, as per the [design doc](https://docs.google.com/document/d/1xxiO5Lv8hV1-zoNoW153dn5CyNky6ZIxqhkogV7smFA)

## How to test this PR locally
   Install the MSSQL 
- Go to Integration UI in Kibana
- Add the Microsoft SQL Server Integration
- Enroll the new elastic-agent
- Test named Instance by providing the named instance port in the Microsoft Sqlserver integration config.
- In the asset -> Transaction Log dashboard you should be able to see the default visualisation.

## Screenshots
